apollo/lib/getWinningTeam.tsx

```diff
@@ -1,10 +1,13 @@
 import { PlayerID } from '@deities/athena/map/Player.tsx';
 import MapData from '@deities/athena/MapData.tsx';
-import { GameEndActionResponse } from '../GameOver.tsx';
+import {
+  GameEndActionResponse,
+  OptionalWinActionResponse,
+} from '../GameOver.tsx';
 
 export default function getWinningTeam(
   map: MapData,
-  actionResponse: GameEndActionResponse,
+  actionResponse: GameEndActionResponse | OptionalWinActionResponse,
 ): 'draw' | PlayerID {
```

Comment:
Ah, this is funky since the function name/module name no longer makes sense now. Maybe we should rename this to `getMatchingTeam` since it gives us the team matching the ActionResponse?

Reply:
This is just my opinion, but I see `processRewards()` is the sole consumer of this `getWinningTeam()` function in the entire codebase, and it checks if the return value is `'draw'` (i.e., `winningTeam !== 'draw'`). 

It just occurred to me that it might be a little awkward to have `matchingTeam !== 'draw'`? In the `processRewards()` perspective, I think it makes more sense to call it `winningTeam` indeed, since the winner takes the rewards?

Maybe we could wait until there are other use cases that calling `getWinningTeam()` wouldn't make sense at all?

Or maybe we could bring `getWinningTeam()` into `processRewards.tsx` as a local function and worry about other use cases later?